extensions/items/standalone.mjs

```diff
@@ -141,11 +156,10 @@ class Standalone {
    */
   async c2s_getAllItems(playerID) {
```

Comment:
Forgot to mention, the first argument for c2s API call is actually a `Player` object. You'll need to do `player.playerID` or `player.getPlayerID()` to get the playerID.